.devops/full.Dockerfile

```diff
@@ -3,7 +3,7 @@ ARG UBUNTU_VERSION=22.04
 FROM ubuntu:$UBUNTU_VERSION as build
 
 RUN apt-get update && \
-    apt-get install -y build-essential python3 python3-pip git
+    apt-get install -y build-essential python3 python3-pip git language-pack-zh-hans language-pack-zh-hant
```

Comment:
Instead of directly modifying the full.Docker,  
I recommend users to extend `ghcr.io/ggerganov/llama.cpp:full` in their own Dockerfile and install these packages.
most of the models introduced in llama.cpp repositories are not fine tuned with zh, but all trained using great amount of en. There's no reason to force all users to include additional language packages in their docker images.

Reply:
If possible, adding a Dockerfile `full-zh.Dockerfile` is a better way, what do you think @DannyDaemonic

Reply:
I'm not against full-zh, but this issue isn't limited to chinese. I'd say we add a bunch of languages to full , and leave light alone.

Reply:
Following up here before I mark this as resolved, but it looks like we don't need to install any language packs to make this work with non-latin languages.